ILSpy.BamlDecompiler/Rewrite/ConnectionIdRewritePass.cs

```diff
@@ -164,14 +167,11 @@ public void Run(XamlContext ctx, XDocument document)
 					{
 						fieldAssignments.Add((section.Labels, field));
 					}
-					else
```

Comment:
I think https://github.com/icsharpcode/ILSpy/pull/2327/files#diff-211b881f639d3ddf7494747c2f9989a31f383ca60a5c3d60e62dcd8cfc1b2266R165-R174 and https://github.com/icsharpcode/ILSpy/pull/2327/files#diff-211b881f639d3ddf7494747c2f9989a31f383ca60a5c3d60e62dcd8cfc1b2266L192-R201 could be extracted to a local function. What do you think?

Reply:
yes, they are same.